memfs/memory_test.go

```diff
@@ -85,6 +85,13 @@ func (s *MemorySuite) TestOrder(c *C) {
 	}
 }
 
+func (s *MemorySuite) TestNotFound(c *C) {
+	files, err := s.FS.ReadDir("asdf")
+	c.Assert(files, HasLen, 0)
+	// JS / wasip have this error message captalised.
+	c.Assert(err, ErrorMatches, "open /asdf: (N|n)o such file or directory")
```

Comment:
Maybe?

```suggestion
	msg := "open /asdf: (N|n)o such file or directory"
	if runtime.GOOS == "windows" {
		msg = "open \\asdf: The system cannot find the file specified."
	}
	c.Assert(err, ErrorMatches, msg)
```

Reply:
Was wondering if this would be problematic on Windows. 🤞